### PR TITLE
Agent orange design mods

### PR DESCRIPTION
**Pull request by kantrowitz**
/disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange/

Enlarged headers
Changed text to "Learn more", bulletized items below

Added piano keys. @gnakm @mariana-almeida @emilyville

### CODE REVIEW

File: _disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange.md [hunk 1]
```diff
@@ -16,128 +38,48 @@ template: 3-panel-filters
...
 
-  <li>
+<br>
 
-    <dl class="panel-list">
-      <dt>Service in Vietnam or Korea</dt>
-      <dd>Between January 9, 1962, and May 7, 1975</dd>
-      <dd><a href="/disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange/service-inside/">Learn more <span class="sr-only">about test and Storage Areas Outside of Vietnam and Agent Orange</span></a></dd>
-    </dl>
+### Were you exposed?
```
Comment (emilyville) [inline _disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange.md]: this seems odd - to end the page in a question. Assuming it's not intentional?

**Comment by emilyville** [comment]
https://vets-website-pr-2258.herokuapp.com/disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange/

I like the version with the boxes better than the piano keys, was there a reason you removed them @kantrowitz ?

**Comment by kantrowitz** [comment]
I included this one as an option.

**Comment by mariana-almeida** [comment]
@emilyville So I think the idea is to revert to old version with boxes, instead of piano keys if we don't like the piano key look? The boxes look better, esp. with the header of "were you exposed?"
This is current version with boxes:
https://www.vets.gov/disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange/

**Comment by kantrowitz** [comment]
The intent was just to provide an option.

On Thu, May 5, 2016 at 6:53 PM, Mariana Almeida <notifications@github.com>
wrote:

> @emilyville <https://github.com/emilyville> So I think the idea is to
> revert to old version with boxes, instead of piano keys if we don't like
> the piano key look? The boxes look better, esp. with the header of "were
> you exposed?"
> This is current version with boxes:
>
> https://www.vets.gov/disability-benefits/conditions/exposure-to-hazardous-materials/agent-orange/
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/department-of-veterans-affairs/vets-website/pull/2258#issuecomment-217303275>
>



-- 
Tiby Kantrowitz
tiby@thesocompany.com
201-401-0133

**Comment by emilyville** [comment]
Thanks for providing an option! let's keep the gray cards. closing.

**Pull request closed without merging**